### PR TITLE
Create info.md

### PR DESCRIPTION
**Pull request by katarinarosiak**
this is totally useless file

### CODE REVIEW

File: info.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Hello world
```
Comment (scogra17) [inline info.md]: LGTM!

**Review by scogra17 — APPROVED** [review]

**Review by tdronkers04 — CHANGES_REQUESTED** [review]
Please add more exclamation points

**Review by mabbason — APPROVED** [review]
I like it! Anything worth doing is worth overdoing...

**Review by scogra17 — APPROVED** [review]

**Review by tdronkers04 — APPROVED** [review]
This is (barely) enough. Will have to do for now

**Pull request merged**